build_windows.md

```diff
@@ -16,17 +16,183 @@ Pandoc could be installed from https://pandoc.org/installing.html When performin
+the ExecutionPolicy on your machine with this commmand:
+
+```
+Get-ExecutionPolicy
+```
+
+If the command returns `Restricted` you should change the ExecutionPolicy to `RemoteSigned`:
+
+```
+Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
+```
+
+This policy allows the execution of local PowerShell scripts, such as `winmake.ps1` for current user:
+
+```
+# Get in the podman git repository directory
+cd podman
+
+# Build podman.exe
+.\winmake.ps1 podman-remote
+
+# Downlaod gvproxy.exe and win-sshproxy.exe
+# that are needed to execute the podman client
+.\winmake.ps1 win-gvproxy
+```
+
+:information_source: To verify that the build got successfull you can check the content of the .\bin\windows` folder. 
+You should see 3 executables:
+
+```
+ls .\bin\windows\
+
+
+    Directory: C:\Users\mario\Git\podman\bin\windows
+
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+-a----         2/29/2024  12:10 PM       10946048 gvproxy.exe
+-a----         2/27/2024  11:59 AM       45408256 podman.exe
+-a----         2/29/2024  12:10 PM        4089856 win-sshproxy.exe
+```
+
+### Configure podman
+
+Create a containers.conf file:
+
+```
+mkdir $env:APPDATA\containers\
+notepad $env:APPDATA\containers\containers.conf
+```
+
+and add the following lines in it:
+
+```toml
+[machine]
+# Specify the provider
+# Values can be "hyperv" or "wsl"
+provider="hyperv"
+[engine]
+# Specify the path of the helper binaries.
+# NOTE: path should use slashes instead of anti-slashes
+helper_binaries_dir=["C:/Users/mario/git/podman/bin/windows"]
+```
+
+Create a policy.json file:
+
+```
+notepad $env:APPDATA\containers\policy.json
+````
+
+and add the following lines in it:
+
+```json
+{
+  "default": [
+    {
+      "type": "insecureAcceptAnything"
+    }
+  ],
+  "transports": {
+    "docker-daemon": {
+      "": [{ "type": "insecureAcceptAnything" }]
+    }
+  }
+}
+```
+
+### Create and start a podman machine
+
+Run a terminal **as an administrator** and execute the following commands to create a podman machine:
+
+```
+.\bin\windows\podman.exe machine init
+```
+
+When `machine init` completes run `machine start`:
+
+```
+.\bin\windows\podman.exe machine start
+```
+
+### Run a container using podman
+
+You are ready to use your locally built podman client for windows to run containers:
+
+```
+.\bin\windows\podman.exe run hello-world
+```
+
+:information_source: Unlike the previous machine commands, this one doesn't require admin privileges.
+
+## Building the Podman client and installer with MSYS2
+
+### Install msys2
 
 Podman requires brew -- a collection of Unix like build tools and libraries adapted for Windows. More details and
 installation instructions are available from their [home page](https://www.msys2.org/). There are also premade GitHub
 actions for this tool that are available.
 
-## Install build dependencies
+### Install build dependencies
```

Comment:
this should be removed.

Reply:
Just this paragraph or the full "Building the Podman client and installer with MSYS2" section?

Reply:
imho, msys2 is NOT required to build on windows; and adding it just confuses things.  Probably others should also weigh in on this notion

Reply:
Agreed. I was careful to keep both options as there was no final decision yet. But I can remove the MSYS2 instructions if that's the consensus.